backend/main/models.py

```diff
@@ -1,3 +1,24 @@
 from django.db import models
 
 # Create your models here.
+class ForestryDistrict(models.Model):
+
+    # forest_manager
+
+    name = models.CharField(max_length=48)
+
+    def __str__(self):
+        return self.name
+
+
+
+class Forestry(models.Model):
+
+    forestry_district = models.ForeignKey(ForestryDistrict, on_delete=models.CASCADE)
+
+    forester = models.IntegerField()
```

Comment:
To pole rozumiem będzie w przyszłości ForeignKey na leśnika zarządzającego danym leśnictwem?